docs/source/nlp/usage_guides/nlp_properties.rst

```diff
@@ -61,6 +61,12 @@ Lexical Density                 Percentage of unique words in the text, rounded
 Unique Noun Count*              Number of unique noun words in the text
 Readability Score               A score calculated based on Flesch reading-ease per text sample. For more information: https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch_reading_ease
 Average Sentence Length         Average number of words per sentence in the text
+Count Unique URLs               Number of unique URLS per text sample.
```

Comment:
We need to make some changes so that our users know how to use the new, non-default properties.
1. Add a new column to this property table, denoting if the property is calculated by default when running `calculate_builtin_properties` or not.
2. In this guide, in the "Including or Ignoring Properties" section, please write about the new mechanism and how to use it to also calculate the new non-default properties on top of the default ones (or also instead of some of them).
3. Please also document this new behaviour in the docstring for `calculate_builtin_properties` both in the text data object and in the text_properties file.

Reply:
Added the details in nlp_properties.rst, text_data_object.rst, and in the docstring. Let me know if any other changes needs to be done.